staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -6368,9 +6368,14 @@ type NamespaceList struct {
 	Items []Namespace `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
 
+// This cannot be added yet:
```

Comment:
I don't expect this will ever be added to this type, since the type is not deprecated (it's used in the binding subresource)... maybe move this comment block to something like a unit test asserting Binding doesn't get this readded (ensuring doesn't have a APILifecycleDeprecated() method)?

Reply:
Comment removed and test added.